### PR TITLE
Added recipe for pyzotero.

### PR DESCRIPTION
**Pull request by moorepants**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages
- [ ] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@conda-forge/help-python

### CODE REVIEW

**Comment by moorepants** [comment]
Interesting the CI passed but with this error present:

```
Building pyzotero with conda-forge/label/main
Setting build arch. This is only useful when pretending to be on another arch, such as for rendering necessary dependencies on a non-native arch. I trust that you know what you're doing.
rendering C:\projects\staged-recipes\recipes\pyzotero for win-64
Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
Exception raised!
Traceback (most recent call last):
  File "C:\projects\staged-recipes\.ci_support\compute_build_graph.py", line 219, in add_recipe_to_graph
    rendered = _get_or_render_metadata(recipe_dir, worker, config=config, finalize=finalize)
  File "C:\Miniconda36-x64\lib\site-packages\conda\exports.py", line 182, in __call__
    return self.func(*args, **kw)
  File "C:\projects\staged-recipes\.ci_support\compute_build_graph.py", line 212, in _get_or_render_metadata
    bypass_env_check=True, config=config, finalize=finalize)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\api.py", line 44, in render
    permit_unsatisfiable_variants=permit_unsatisfiable_variants)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\render.py", line 809, in render_recipe
    allow_no_other_outputs=True, bypass_env_check=bypass_env_check)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\render.py", line 723, in distribute_variants
    (mv, need_source_download, None)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\metadata.py", line 1314, in dist
    return '%s-%s-%s' % (self.name(), self.version(), self.build_id())
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\metadata.py", line 1297, in build_id
    out = build_string_from_metadata(self)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\metadata.py", line 557, in build_string_from_metadata
    for ms in metadata.ms_depends('run'):
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\metadata.py", line 1212, in ms_depends
    sys.exit(msg)
SystemExit: Error: bad character '>=' in package version dependency 'feedparser'
Perhaps you meant 'feedparser >=5.1.0'
invalid recipe dir: C:\projects\staged-recipes\recipes\pyzotero - skipping
Computed that there are 0 distributions to build from 1 recipes
Nothing to do
mkdir conda_packages
Collecting artifacts...
No artifacts found matching 'conda_packages\*.bz2' path
Build success
```

**Comment by moorepants** [comment]
@conda-forge/help-python I believe this should be ready for merging.

File: recipes/pyzotero/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "Pyzotero" %}
```
Comment (xhochy) [inline recipes/pyzotero/meta.yaml]: Is the captialization intended?

Comment (moorepants) [reply recipes/pyzotero/meta.yaml]: That's how the author named it.

**Pull request merged**